package.json

```diff
@@ -5,7 +5,7 @@
   "description": "Wait for the specified jobs in the same run to complete successfully before proceeding",
   "main": "dist/index.js",
   "scripts": {
-    "build": "rm -rf dist bin && ncc build --source-map --license licenses.txt -o dist src/index.ts",
```

Comment:
Could you revert this? this to make sure `Check dist/` workflow validates correct files

Reply:
This is fixed; thanks for catching that.  I had to turn it off temporarily.